lib/claper_web/views/components/input_component.ex

```diff
@@ -37,6 +37,39 @@ defmodule ClaperWeb.Component.Input do
     """
   end
 
+  def textarea(assigns) do
+    assigns =
+      assigns
+      |> assign_new(:required, fn -> false end)
+      |> assign_new(:autofocus, fn -> false end)
+      |> assign_new(:placeholder, fn -> false end)
+      |> assign_new(:readonly, fn -> false end)
+      |> assign_new(:labelClass, fn -> "text-gray-700" end)
+      |> assign_new(:fieldClass, fn -> "bg-white" end)
+      |> assign_new(:value, fn -> input_value(assigns.form, assigns.key) end)
+
+    ~H"""
+    <div class="relative">
+      <%= label(@form, @key, @name, class: "block text-sm font-medium #{@labelClass}") %>
+      <div class="mt-1">
+        <%= text_input(@form, @key,
```

Comment:
@alxlion I made it as a `text_input` instead of `textarea` here. When I try `textarea` the user input value are not been updated not sure what the issue was though.

![image](https://github.com/ClaperCo/Claper/assets/43700516/c385855b-4831-452e-a189-b764f60113f1)